smalltalksrc/VMMaker/DruidJIT.class.st

```diff
@@ -7129,7 +7954,7 @@ DruidJIT >> gen_StoreAndPopReceiverVariableBytecode0 [
 	jump5 := self JumpNonZero: 0.
 	self MoveR: t1 R: t2.
 	self CmpCq: objectMemory getMemoryMap getNewSpaceStart R: t2.
-	jumpTrue := self JumpGreaterOrEqual: 0.
+	jumpTrue := self JumpAboveOrEqual: 0.
```

Comment:
I wonder if here having the unsigned comparison is ok.
I think it does because we are comparing addresses...
This means that before it was wrong :)

Reply:
Yes, there was a bug in the long method where jumps check types. I thought that you have it fixed also in your branch.